PendingReleaseNotes.md

```diff
@@ -8,14 +8,9 @@ v1.8...
 
 ## Breaking Changes
 
-### Ceph
+- Flex driver is fully deprecated. If you are still using flex volumes, before upgrading to v1.8
+  you will need to convert them to csi volumes. See the flex conversion tool.
```

Comment:
suggestion: add a link to the tool.

Reply:
In the final release notes we can revise this and add a link. This is really just a placeholder for now.